web-dapp/server-lib/send_response.js

```diff
@@ -6,7 +6,9 @@ function send_response(res, obj) {
         obj.x_id = res.req.x_id;
     }
     logger.log('[response] (' + res.req.log_prfx + ') ' + JSON.stringify(obj));
-    return res.json(obj);
+    const status = (obj.ok) ? 200 : 400;
```

Comment:
Is there any other way to test, without setting response code?
Reason: error handling by DApp is changed - at the present moment if obj.ok is false, DApp displays a detailed error message. However if status is changed from 200, another handler will be used, that just says there was a server error